README.md

```diff
@@ -26,16 +26,16 @@ in your file
 ## `AlAwsCollector`
 Base class for AWS lambda based collectors
 
-@param {Object} context - context of Lambda's function.
-@param {string} collectorType - collector type (cwe as example).
-@param {string} ingestType - ingest data type (secmsgs, vpcflow, etc).
-@param {string} version - version of collector.
-@param {Object} aimsCreds - Alert Logic API credentials.
-@param {string} [aimsCreds.access_key_id] - Alert Logic API access key id.
-@param {string} [aimsCreds.secret_key] - Alert Logic API secret key.
-@param {function} formatFun - callback formatting function
-@param {Array.<function>} healthCheckFuns - list of custom health check functions (can be just empty, so only common are applied)
-@param {Array.<function>} statsFuns - list of custom stats functions (can be just empty, so only common are applied)
+* @param {Object} context - context of Lambdas function.
+* @param {string} collectorType - collector type (cwe as example).
+* @param {string} ingestType - ingest data type (secmsgs, vpcflow, etc).
+* @param {string} version - version of collector.
+* @param {Object} aimsCreds - Alert Logic API credentials.
+* @param {string} [aimsCreds.access_key_id] - Alert Logic API access key id.
+* @param {string} [aimsCreds.secret_key] - Alert Logic API secret key.
+* @param {function} formatFun - callback formatting function
+* @param {Array.<function>} healthCheckFuns - list of custom health check functions (can be just empty, so only common are applied)
+* @param {Array.<function>} statsFuns - list of custom stats functions (can be just empty, so only common are applied)
 
 # Debugging
 
```

Comment:
MayI ask youto remove `# Known Issues/ Open Questions` section

Reply:
you may.